BeagleBone_black/src/motor_control/motor.hpp

```diff
@@ -31,10 +31,10 @@ using utils::Logger;
 namespace motor_control {
 // Contains the RPM of each of the motors
 struct MotorsRpm {
-  int32_t rpm_FL;
-  int32_t rpm_FR;
-  int32_t rpm_BL;
-  int32_t rpm_BR;
+  int32_t rpm_FL_;
+  int32_t rpm_FR_;
+  int32_t rpm_BL_;
+  int32_t rpm_BR_;
 };
 
 class Motor {
```

Comment:
Add a class comment explaining what the `Motor` class is. Does it represent a single physical motor? What can it do?